test/GoodGhosting_Funds_Redeemed_From_Contract_On_EarlyWithdraw.test.js

```diff
@@ -146,8 +173,8 @@ contract("GoodGhosting", (accounts) => {
         });
 
         it("players withdraw from contract", async () => { // having test with only 1 player for now
-            // starts from 1, since player1 (loser), requested an early withdraw
-            for (let i = 1; i < players.length; i++) {
+            // starts from 2, since player1 (loser) and player2 (secondLoser), requested an early withdraw
+            for (let i = 2; i < players.length; i++) {
```

Comment:
depending on the comment above about `player1` doing an early withdraw or not, we may need to change this loop to start from 0 and ignore player 2 (in case player 1 does not early withdraw, but just loses the game because player1 stopped paying).

Reply:
basically, index 0 is player 1 and so on so hence player 1 and player 2 (index 1) loose

Reply:
Yeah, it makes sense now that I saw player 1 also loses. Thanks for explaining it.